packages/mrm-task-eslint/index.js

```diff
@@ -8,6 +8,29 @@ const {
 	getExtsFromCommand,
 } = require('mrm-core');
 
+const normalizePresetPackageName = presetName => {
+	const prefix = 'eslint-config';
+	const presetNameRegex = /^(?:(@[^/]+)\/?)?((?:eslint-config-)?[^/]*)(?:\/[^/]+)?$/;
+	const match = presetName.match(presetNameRegex);
+
+	if (!match) {
+		throw new Error('Invalid preset name');
```

Comment:
Let's make it more clear for the user:

```suggestion
		throw new Error(`Invalid preset name is passed to the eslint task: ${presetName}`);
```